README.md

```diff
@@ -96,13 +96,20 @@ Then, open a command prompt in the new `ixmp` directory and type
 
 ## Configure Model Files
 
-By default, the model files are stored with your installation. If you would like
-to store them some place locally, open a command prompt and type
+By default, the model files are stored with your installation. Many users will
+simply want to run MESSAGEix, and will never need to see the these files;
+however, some users will want to edit the files directly to add new parameters,
```

Comment:
I'd rephrase that to "new types of parameters" to reduce (but not eliminate, unfortunately) the confusion between a parameter and its elements. Or maybe say "change the mathematical formulation"?